components/PlayersList/index.js

```diff
@@ -0,0 +1,27 @@
+import { players } from "@/lib/data";
+import PlayerCard from "@/components/PlayerCard";
+import styled from "styled-components";
+
+export default function PlayersList() {
+  return (
+    <>
+      <h1>Players overview</h1>
+      <StyledPlayersList>
+        {players.map((player)=>(
+          <PlayerCard
+            key={player.slug}
+            player={player} />
+        ))}
+      </StyledPlayersList>
+    </>
+  )
+}
+
+const StyledPlayersList = styled.div`
```

Comment:
```suggestion
const StyledPlayersList = styled.ul`
```

this should be a list to be more semantic